browser/.beforeprettier

```diff
@@ -44,7 +44,7 @@
 /src/control/Control.SearchBar.js
 /src/control/Control.Selection.js
 /src/control/Control.SheetsBar.js
-/src/control/Control.Sidebar.js
+/src/control/Control.Sidebar.ts
```

Comment:
Please remove this, beforeprettier should be only used for old files.
We want to force new styling in the new files